src/styles/utils/_keyframes.scss

```diff
@@ -0,0 +1,29 @@
+@keyframes sek {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes min {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes hour {
```

Comment:
The 'hour' keyframe is not needed for a stopwatch with minute and second hands. Consider removing it.